### PR TITLE
[DASH-2409] Move app MobileOnboarding to its own component

### PR DESCRIPTION
**Pull request by wpasinski**

### CODE REVIEW

File: src/common/MobileOnboarding/AppMobileOnboarding.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
```
Comment (patrykkopycinski) [inline src/common/MobileOnboarding/AppMobileOnboarding.js]: remove it

File: src/common/MobileOnboarding/AppMobileOnboarding.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import { RaisedButton } from 'material-ui';
+
```
Comment (patrykkopycinski) [inline src/common/MobileOnboarding/AppMobileOnboarding.js]: remove it

File: src/assets/sass/_base.sass [hunk 3]
```diff
@@ -56,6 +52,10 @@ input:-webkit-autofill
   svg
     display: none !important
 
+.app
```
Comment (patrykkopycinski) [inline src/assets/sass/_base.sass]: as `#app` is necessary there is no point to create a separate class to style it

File: src/pages/app.jsx [hunk 4]
```diff
@@ -62,123 +78,38 @@ const App = React.createClass({
...
-        minWidth={768}
-        widthMethod="componentWidth"
-        style={styles}
-      >
-        {this.props.children}
-        <SnackbarNotification />
-      </Breakpoint>
-    );
-  },
+  renderHelmet() {
```
Comment (patrykkopycinski) [inline src/pages/app.jsx]: do we need a separate method to render a single component?
(comment isn't necessary i removed this dependency in #205

File: src/common/MobileOnboarding/AppMobileOnboarding.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
```
Comment (patrykkopycinski) [inline src/common/MobileOnboarding/AppMobileOnboarding.js]: this component shouldn't bee in `common` as it's component filled with end-content

File: src/assets/sass/_base.sass (hunk 3 above)
Comment (wpasinski) [inline src/assets/sass/_base.sass]: we should not style via `id` selector no matter what.

Comment (patrykkopycinski) [inline src/assets/sass/_base.sass]: in general we should, but it isn't a case

**Pull request merged**